server/schema/departments.schema.ts

```diff
@@ -13,7 +13,7 @@ import {
 export const departments = pgTable('departments', {
   id: smallserial('id').primaryKey(),
   name: varchar('name', { length: 64 }).notNull(),
-  alias: char('alias', { length: 2 }).notNull(),
+  alias: char('alias', { length: 4 }).notNull(),
```

Comment:
We will change this later if necessary. Aliases should be only 2 chars.

Reply:
For some, like Chemistry and Electronics, they were requiring 3-4 letters, thus I changed it. Will revert back